docs/source/pillars/computing_technology.md

```diff
@@ -33,8 +33,8 @@ _The interfaces used for interacting with the TRE management system and the TRE
     In practice a web browser is the most common way to achieve this.
   - Recommended
 * - A TRE workspace should provide an environment familiar to the users of the TRE.
-  - This may be in the form of a virtual Windows or Linux desktops, web applications, or a terminal.
-    The use of custom developed TRE-specific software should be avoided when widely used open-source alternatives already exist.
+  - For TREs where the primary researchers are data scientists, this may take the form of a virtual Windows or Linux desktops, non-desktop interfaces such as JupyterLab and other web applications, or a terminal.
```

Comment:
How about removing the "where the primary researchers are data scientists" and giving an example of what the OpenSAFELY interface looks like?